packages/google_maps_flutter/google_maps_flutter/README.md

```diff
@@ -32,84 +41,82 @@ For more details, see [Getting started with Google Maps Platform](https://develo
 
 1. Set the `minSdkVersion` in `android/app/build.gradle`:
```

Comment:
You can just remove this whole bullet point while you are touching this; Flutter itself only supports 21+ now.

Reply:
Done.